packages/components/src/utils/config-values.js

```diff
@@ -48,7 +48,7 @@ export default {
 	borderWidth: '1px',
 	borderWidthFocus: '1.5px',
 	borderWidthTab: '4px',
-	spinnerSize: '18px',
+	spinnerSize: '18',
```

Comment:
Just for my own curiosity, why is it unitless?

Reply:
In order for the spinner to scale correctly it's used on attributes like the svg viewbox, and the circle radius. I think the radius would accept a unit, but unfortunately the viewbox does not.

Reply:
Since it's unitless, should be change it to be a number instead of a string?

Reply:
Marking as resolved, in favour of https://github.com/WordPress/gutenberg/pull/37551#discussion_r788614818